wasmplugin/plugin.go

```diff
@@ -592,21 +593,6 @@ func (ctx *httpContext) OnHttpStreamDone() {
 	tx := ctx.tx
 
 	if tx != nil {
-		if !tx.IsRuleEngineOff() && !ctx.interruptedAt.isInterrupted() {
```

Comment:
Ping @anuraaga

Reply:
I think I did it just to be symmetric with processrequest. While interruptions can't be generated here, running in detectiononly mode or similar with phase 4 rules that apply to bodyless responses could still need it? Maybe not as much if multiphase can handle the rules, but it won't be able to handle all rules I guess